src/encoding.ts

```diff
@@ -98,6 +98,11 @@ export interface Encoding<F> {
    */
   tooltip?: FieldDefWithCondition<TextFieldDef<F>> | ValueDefWithCondition<TextFieldDef<F>>;
 
+  /**
+   * A URL to load upon mouse click.
+   */
+  href?: FieldDefWithCondition<TextFieldDef<F>> | ValueDefWithCondition<TextFieldDef<F>>;
```

Comment:
It shouldn't be TextFieldDef?

Reply:
What if you want a conditional url?

Reply:
Well, TextFieldDef and conditional are orthogonal.  TextFieldDef has `format` that href shouldn't

Reply:
Fixed and rebased.